docs/topics/testing/advanced.txt

```diff
@@ -705,6 +705,12 @@ Methods
     Computes and returns a return code based on a test suite, and the result
     from that test suite.
 
+.. method:: DiscoverRunner.log(msg, level=None)
+
+    Logs a message at the given `logging level <https://docs.python.org/3/library/logging.html#levels>`_
+    respecting the current ``verbosity``. For example, INFO will be logged if the verbosity is at least 1,
```

Comment:
Backticks are needed around `INFO`, `1`, `DEBUG`, and `2`.

Reply:
Please also add a `.. versionadded 4.0` directive. With a shirt note saying this method was added. 

https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features